### ISSUE TITLE
Export full_path when a resource name isn't its full path

### ISSUE THREAD

**Issue by pcfens** [F5Networks/terraform-provider-bigip#490]
### Is your feature request related to a problem? Please describe.
Some resource names don't match their full path names, even though other resources require a full path (e.g. ssl profiles require the full path to certs and keys, but certs and keys don't export this info)

### Describe the solution you'd like
For something like this to work:
```terraform
resource "bigip_ltm_profile_client_ssl" "example_com" {
  name          = "/Common/example.com"
  defaults_from = "/Common/clientssl"

  cert  = bigip_ssl_certificate.example_com.full_path
  key   = bigip_ssl_key.example_com.full_path
}

resource "bigip_ssl_certificate" "example_com" {
  name      = "example.com"
  content   = file("ssl/certs/example.com.crt")
  partition = "Common"
}

resource "bigip_ssl_key" "example_com" {
  name      = "example.com"
  content   = file("ssl/certs/example.com.key")
  partition = "Common"
}
```

### Describe alternatives you've considered
Manually concatenating to find the path isn't hard, but seems unnecessary since it can be a computed field for consistency across other Terraform providers.

Another possible change would be to keep naming consistent and always require the partition name in the name, though this would be a breaking change, while exporting would work as is.

**Comment by trinaths** [comment]
@pcfens - Is that the `name` in resource `bigip_ltm_profile_client_ssl` an issue  OR `cert` and `key` attributes ? Please share more clear info.

**Comment by pcfens** [comment]
It's the cert and key attributes in `bigip_ltm_profile_client_ssl` that require the full path. They aren't exported by the `bigip_ssl_certificate` and `bigip_ssl_key` resources so Terraform can't automatically build dependency graphs.

**Comment by trinaths** [comment]
created JIRA INFRAANO-399 for internal PM tracking.

**Comment by RavinderReddyF5** [comment]
@pcfens fixed in v1.11.0
